src/components/App.js

```diff
@@ -25,6 +26,7 @@ const App = () => (
       <Route component={() => <PasswordForgetPage />} exact path={routes.PASSWORD_FORGET} />
       <Route component={() => <HomePage />} exact path={routes.HOME} />
       <Route component={() => <AccountPage />} exact path={routes.ACCOUNT} />
+      <Route component={() => <Users />} exact path={routes.USERS} />
```

Comment:
Just a tiny advice: if you use `component` prop in react router you can directly pass component class to it.
Example:
```jsx
<Route component={Users} exact path={routes.USERS} />
```